### ISSUE TITLE
TextEditor: Missing ui-state-error when invalid

### ISSUE THREAD

**Issue by melloware** [primefaces/primefaces#8663]
**Describe the defect**
TextEditor is missing `ui-state-error` and `aria-required` and `aria-invalid` properties 

**Environment:**
 - PF Version: _11.0.0_
 - PF Theme: _ALL_
 - JSF + version: _ALL_
 - Affected browsers: _ALL_

**To Reproduce**
1. Make a TextEditor `required="true"`
2. Submit the empty TextEditor
3. Notice none of the attributes are present.

cc @fcorneli

**Comment by melloware** [comment]
Linked to Theme ticket: https://github.com/primefaces/primefaces/issues/8661

**Comment by fcorneli** [comment]
Tested out 12.0.0-SNAPSHOT. `ui-state-error` indeed gets added correctly, but this probably needs some additional styling as stated by #8661

**Comment by melloware** [comment]
thanks for testing!  Yep you can copy those styles into your app for now but hopefully Mert will merge them into the themes before 12.0.0 final!
